AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -109,17 +194,36 @@ public final class PayToComponent: PaymentComponent,
         )
         formViewController.title = paymentMethod.displayInformation(using: configuration.localizationParameters).title
         formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
         formViewController.append(flowSelectionTitleLabelItem.padding())
         formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
         formViewController.append(flowSelectionItem.padding())
+        formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
+        formViewController.append(identifierPickerItem.padding())
+        formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
+        payIDContentView(formVC: formViewController)
 
         if configuration.showsSubmitButton {
             formViewController.append(FormSpacerItem(numberOfSpaces: 2))
-            formViewController.append(continueButton)
+            formViewController.append(continueButtonItem)
         }
 
         return formViewController
     }()
+
+    // MARK: - Private
+
+    private func payIDContentView(formVC: FormViewController) {
```

Comment:
The function still does not really say what it does - based on the name I'd assume it returns a contentView but it actually appends item to the form.

Maybe instead call it `func appendNameInputItems(to formVC: FormViewController)` or something similar to not have to go through the function implementation to figure out what it does.

Reply:
I have added this function to separate static and dynamic UI content (will be added in later PRs). This function will call methods for static and dynamic UI.

Reply:
I have renamed the method to `appendItemsTo(formVC: FormViewController)`